src/infra/brick.go

```diff
@@ -173,10 +177,6 @@ func (brick *Brick) Enrich(bcy BrickConfYaml, infra *Infra) error {
 func (b *Brick) CreateFormatters() (fileFormatters map[string]Formatter, env_formatters EnvFormat, err error) {
 	fileFormatters = make(map[string]Formatter)
 
-	// NOTE(half-shell): This is pretty ugly, but there does not seem to be a way to create
-	// a 3-fold nested map otherwise.	// This is a pretty good use case for a tree-like structure!
-	varNameToVal := make(map[string]interface{})
-	pathToVars := make(map[string]map[string]interface{})
 	// Temporary variable holding the values dispatched by format, path and variable name.
 	// e.g. rawInputs[<data_format>][<file_path>][<variable_name>] => <variable_value>
 	rawInputs := make(map[InputFormat]map[string]map[string]interface{})
```

Comment:
Je garderai bien le com' concernant une structure appropriée pour ce cas de figure quand même
```suggestion
	// Temporary variable holding the values dispatched by format, path and variable name.
	// e.g. rawInputs[<data_format>][<file_path>][<variable_name>] => <variable_value>
	// NOTE: This is a pretty good use case for a tree-like structure!
	rawInputs := make(map[InputFormat]map[string]map[string]interface{})
```

Reply:
comme ça c'est mieux ou tu veux tous le com ? Je sais pas si c'est toujours ugly, même si c'est chiant à utiliser sachant qu'il faut tester si toutes les sous key existent.